js/index.js

```diff
@@ -80,6 +82,14 @@ exports = new (Class(function () {
         });
     });
 
+    pluginOn('SettingsOpened', function () {
+      invokeCallbacks(cb_notif_enable_popup);
```

Comment:
i understand this event. dont we need event when user enables?? better to have it